qemu/tests/block_during_io.py

```diff
@@ -140,5 +141,14 @@ def run_bg_test(target, args=(), kwargs={}):
     bg_test_thread.join(bg_test_thread_timeout)
 
     if not shutdown_vm:
+        if with_data_disks and windows:
+            # XXX: The data disk letters will be changed after reboot in windows.
```

Comment:
Just \"system_reset\", hit this issue, please change \"reboot\" to \"system_reset\".